react/Section/Section.less

```diff
@@ -0,0 +1,19 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  padding: (@grid-row-height * 2) @grid-gutter-width;
+  @media only screen and (min-width: 980px) {
```

Comment:
We have `980px` here and `1000px` for `PageBlock`, should these have a common value?